content/posts/enable-beta-addons.md

```diff
@@ -0,0 +1,134 @@
+
+
+In the last year, we have been focusing ourselves on providing a better experience for authors and readers of the documentation.
+Today, we are thrilled to introduce our latest feature,
+designed around user feedback to build a better user experience: **Read the Docs Addons**.
+
+Read the Docs Addons helps authors to keep their documentation updated by focusing on the most relevant pages,
+while simplifying the review process --and also helps readers to navigate the documentation in a simple way,
+always knowing the versions they are reading avoiding confusions.
+
+Continue reading to explore the full list of addons and how to enable them in your project.
+
+## How to enable addons in your project
+
+To enable Read the Docs Addons in your project follow these steps:
+
+1. Go to the [new beta dashboard](https://beta.readthedocs.org).
+1. Click on the project you want to enable addons.
+1. Go to **Settings**, then **Addons (Beta)** from the left bar.
+1. Check **Enable Addons**.
+
+Now you have Read the Docs Addons enabled,
+readers of your documentation can start using these features.
+
+
+## Full list of addons
+
+
+### Analytics
+
+![Analytics Addons](/images/addons-analytics.png)
+
+Analytics allows authors to explore your documentation traffic to understand what are the pages readers read the most.
+With this data, your team is able to **prioritize the work on pages with high traffic** over others with less.
+Besides, you can find out what are the terms your users search the most and
+write documentation pages that include those terms so they can easily find them.
+
+### DocDiff
+
+![Docdiff Addons](/images/addons-docdiff.gif)
+
+DocDiff shows changes made on the rendered versions by **visually highlighting the differences between
+the current pull request and the latest version of the project's documentation**.
+This addons allows reviewers to quickly find what has changed on this pull request
+but also find style/rendering issues before accepting and merging the pull request.
+
+### Pull request notification
+
+![Pull request notification](/images/addons-pull-request-notification.png)
+
+This notification warns users letting them know the documentation page they are reading
+comes from a pull request to clearly distinguish this version from the official one.
+
+
+### Flyout
+
+![Flyout Addons](/images/addons-flyout.png)
+
+The _flyout_ is the small floating Read the Docs section where all the documentation versions and translations are listed.
+This allows readers to **find the exact version they are looking for** and also read the documention in their own language if it's available.
+Besides, it contains useful links to go to the project's home, builds page and offline formats such as PDFs and ebooks.
+
+### Non-stable notification
```

Comment:
I feel like we're highlighting notifications in 3 sections... shouldn't we just combine them together and note that it supports each type as a bullet or similar? Seems like a lot of notifications.

Reply:
I don't have a strong opinion here. I think it's fine for now, and I like that we are showing them all in the screenshots with different doctools on each. We can find a different/better pattern for our own documentation if we want.